src/components/ProfileTree/ProfileLabel.js

```diff
@@ -133,10 +147,14 @@ export const ProfileLabel = memo(({
 });
 
 ProfileLabel.propTypes = {
-  entityKey: PropTypes.string.isRequired,
-  recordData: PropTypes.object.isRequired,
   linkingRules: PropTypes.object.isRequired,
+  recordData: PropTypes.object.isRequired,
```

Comment:
Non-primitive data types like `object` would be great to replace with more detailed data types like `shape` or `exact`. See [docs](https://reactjs.org/docs/typechecking-with-proptypes.html) for more details.